packages/gestalt/src/Avatar.js

```diff
@@ -77,15 +28,8 @@ export default class Avatar extends React.PureComponent<AvatarProps, State> {
     verified: PropTypes.bool,
   };
 
-  state = {
-    isImageLoaded: true,
-  };
-
-  handleImageError = () => this.setState({ isImageLoaded: false });
-
   render() {
     const { name, outline, size, src, verified } = this.props;
-    const { isImageLoaded } = this.state;
     const width = size ? sizes[size] : '100%';
     const height = size ? sizes[size] : '';
     return (
```

Comment:
@bradencanderson do you think there is any way we can avoid this top level `<Box/>` duplication since `<GroupAvatar/>` also supports an outline and it looks like this Box mostly sets outlines and sizing (which we can consume into `GroupAvatar` since the sizings and outline design are supposed to be the same.

Reply:
Thanks @pfarejowicz, I didn't notice both of them support `outline`. I'll update.